packages/dds/ordered-collection/package.json

```diff
@@ -75,7 +75,7 @@
     "@fluidframework/build-tools": "^0.2.71273",
     "@fluidframework/eslint-config-fluid": "^0.28.2000",
     "@fluidframework/mocha-test-setup": "^2.0.0",
-    "@fluidframework/ordered-collection-previous": "npm:@fluidframework/ordered-collection@^0.59.0",
+    "@fluidframework/ordered-collection-previous": "npm:@fluidframework/ordered-collection@^1.0.0",
```

Comment:
We need to sort out auto-integration story here. Every time we do main-to-next integration, we are picking up "previous" from main, that may not be correct.